README.md

```diff
@@ -0,0 +1,23 @@
+# Doula Cooperative of Rochester
+ 
+ Site: https://www.doulacooperative.org
+
+ ## Getting Started
+ ```
+ $ npm run tools
+ $ firebase login
+ $ yarn install
+ $ yarn serve
+ ```
+
+ ## Modify the Doula List
+ 1. Update: src/templates/partials/_doulas.pug
```

Comment:
`0. git checkout -b [doula-name]` where doula-name is just the name of the doula in [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Special_case_styles)

Reply:
I'm not planning to have Cheryl work with source control. The idea is:
1. I'll checkout the source and set her branch/working dir state
2. She'll make edits
3. I'll commit then pull from my pc
4. I'll test
5. I'll PR